opentok/opentok-ios-sdk-samples
# [VIDCS-922] Add Video transformers sample

**What is this PR doing?**

Adds Video transformers sample.

**How should this be manually tested?**

There is a button which toggles the video transformers (background blur and logo).
Once set, it should be visible instantly on the video stream.

FYI: On the simulator the transformers do not appear on the publisher's view. Only the subscriber can see the transformers being applied to the system.

https://jira.vonage.com/browse/VIDCS-922
